script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -273,7 +273,7 @@ A human cannot pass through a shower of debris on his own.
 [sha:f6dc95af6904944bd4332a85c1d88744049ea3db]{
 -- Page 674, Offset 10471.
 -- 真祖の姫による援護を満足に活用する事もできない。
-He cannot even make use of support from the Princess of the True Ancestors to that end.
+He cannot even make use of the support from the Princess of the True Ancestors to that end.
```

Comment:
Not sure I like the here. Either `make use of support from` or `make use of the support of`